resourcecontrollerv2/resource_controller_v2_integration_test.go

```diff
@@ -1388,8 +1444,8 @@ var _ = Describe("Resource Controller - Integration Tests", func() {
 		// 	result, resp, err := service.GetResourceInstance(options)
 
 		// 	//printing info for debugging
-		// 	fmt.Printf("\nDEBUGGING - testReclaimInstanceGuid: %s\n", testReclaimInstanceGuid)
-		// 	fmt.Printf("\nDEBUGGING - Transaction-Id: rc-sdk-go-test53-%s\n", transactionId)
+		// 	fmt.Fprintln(GinkgoWriter, "\nDEBUGGING - testReclaimInstanceGuid: %s\n", testReclaimInstanceGuid)
```

Comment:
You can probably just leave these uncommented since the messages only get printed in "verbose" mode.